src/lambda/api/authorizer.ts

```diff
@@ -241,6 +241,31 @@ class PersonaAuthorizer {
 
     return this.personaClient;
   }
+
+  pathMatch(pathDefination: string, path: string): boolean {
+    const pathDefinationParts = pathDefination.split("/");
+    const pathParts = pathDefination.split("/");
```

Comment:
Looks like a mistake:

```suggestion
    const pathParts = path.split("/");
```

Which makes me think we're missing some test coverage?

Reply:
Done